core-web/libs/edit-content/src/lib/pipes/contentlet-status.pipe.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DotMessageService } from '@dotcms/data-access';
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
+
+import { ContentletStatusPipe } from './contentlet-status.pipe';
+
+describe('ContentletStatusPipe', () => {
+    let pipe: ContentletStatusPipe;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            providers: [
+                ContentletStatusPipe,
+                { provide: DotMessageService, useValue: { get: (arg) => arg } }
+            ]
+        });
+        pipe = TestBed.inject(ContentletStatusPipe);
```

Comment:
should we use spectator? https://ngneat.github.io/spectator/docs/testing-directives/

Reply:
We can, but with spectator is more like a integration testing

```
    const createPipe = createPipeFactory({
        pipe: ContentletStatusPipe,
        template: `{{ (contentlet | contentletStatus).label }}`,
        providers: [{ provide: DotMessageService, useValue: { get: () => 'Published' } }]
    });

    it('should return the correct label', () => {
        spectator = createPipe({
            hostProps: {
                contentlet: {
                    live: true,
                    working: false
                }
            }
        });
        expect(spectator.element).toHaveText('Published');
    });
```

We need to embed the pipe and create a host, that test is already on the aside component. I think is better to test this pipe like a "class"
opinions? cc: @fmontes @rjvelazco @zJaaal